src/containers/PfSpinnerContainer.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { KialiAppState } from '../store/Store';
+import { connect } from 'react-redux';
+
+type PfSpinnerProps = {
+  isLoading: boolean;
+};
+
+const mapStateToProps = (state: KialiAppState) => ({
+  isLoading: state.globalState.isLoading
+});
+export const PfSpinner: React.SFC<PfSpinnerProps> = props => {
```

Comment:
if this really isn't a PatternFly spinner, and if the implementation is independent of Patternfly, should we rename this container and the exported variables so they don't have "Pf" in the names?

Reply:
Good point. I need to think on this one.